src/js/components/Meter/__tests__/__snapshots__/Meter-test.js.snap

```diff
@@ -287,58 +287,58 @@ exports[`Meter many values 1`] = `
       strokeWidth={24}
     />
     <path
-      d="M 336,12 L 384,12"
```

Comment:
I wouldn't expect any snapshot changes to existing tests for this.

Reply:
I am not sure why this changed.